tests/Unit/CommandLine/Commands/Checks/ForbiddenChangesCommandTest.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace Zooroyal\CodingStandard\Tests\Unit\CommandLine\Commands\Checks;
+
+use Hamcrest\Matcher;
+use Hamcrest\MatcherAssert;
+use Hamcrest\Matchers as H;
+use Mockery;
+use Mockery\MockInterface;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Zooroyal\CodingStandard\CommandLine\Commands\Checks\ForbiddenChangesCommand;
+use Zooroyal\CodingStandard\CommandLine\Commands\StaticCodeAnalysis\FindFilesToCheckCommand;
+use Zooroyal\CodingStandard\CommandLine\FileFinders\DiffCheckableFileFinder;
+use Zooroyal\CodingStandard\CommandLine\Library\Environment;
+use Zooroyal\CodingStandard\CommandLine\ValueObjects\GitChangeSet;
+use Zooroyal\CodingStandard\Tests\Tools\SubjectFactory;
+
+class ForbiddenChangesCommandTest extends TestCase
+{
+    /** @var ForbiddenChangesCommand */
+    private $subject;
+    /** @var MockInterface[] */
+    private $subjectParameters;
+    /** @var string */
+    private $blacklistToken = '.dontChangeFiles';
+    /** @var string */
+    private $whitelistToken = '.doChangeFiles';
+
+    protected function setUp()
+    {
+        $subjectFactory = new SubjectFactory();
+        $buildFragments = $subjectFactory->buildSubject(ForbiddenChangesCommand::class);
+        $this->subject = $buildFragments['subject'];
+        $this->subjectParameters = $buildFragments['parameters'];
+    }
+
+    protected function tearDown()
+    {
+        Mockery::close();
+        parent::tearDown();
+    }
+
+    /**
+     * @test
+     */
+    public function configure()
+    {
+        /** @var MockInterface|FindFilesToCheckCommand $localSubject */
+        $localSubject = Mockery::mock(ForbiddenChangesCommand::class)->makePartial();
+
+        $localSubject->shouldReceive('setName')->once()->with('checks:forbidden-changes');
+        $localSubject->shouldReceive('setDescription')->once()
+            ->with('Checks for unwanted code changes.');
+        $localSubject->shouldReceive('setHelp')->once()
+            ->with(
+                'This tool checks if there where changes made to files. If a parent directory contains a '
```

Comment:
den help string zu checken halt ich für'n bisschen overkill, aber okay, your funeral ;)

Reply:
Ja ich war gerade in Stimmung :D